src/input/Keyboard.cc

```diff
@@ -1194,7 +1194,8 @@ bool Keyboard::processKeyEvent(EmuTime::param time, bool down, const KeyEvent& k
 {
 	auto mode = keyboardSettings.getMappingMode();
 
-	auto keyCode  = keyEvent.getKeyCode();
+	auto keyCode = keyEvent.getKeyCode();
+	auto ScanCode = keyEvent.getScanCode();
```

Comment:
we prefer to start a local variable with lower case... but I don't mind to change this myself before merging if I find out how :)